src/framework/global/types/number.h

```diff
@@ -0,0 +1,176 @@
+#ifndef MU_GLOBAL_NUMBER_T_H
+#define MU_GLOBAL_NUMBER_T_H
+
+#include <limits>
+#include <cassert>
+#include <cmath>
+
+#include "global/logstream.h"
+
+namespace mu {
+// default
+constexpr double COMPARE_REAL_EPSILON(1000000.0);
```

Comment:
Actually Epsilon is maybe not a great name for such a large number :smile:
Isn't it actually rather `COMPARE_REAL_ONE_OVER_EPSILON`, i.e. $\displaystyle\frac{1}{\epsilon}$?